frontend/src/App.js

```diff
@@ -1,9 +1,29 @@
-import React from 'react'
+import React, {useState} from 'react'
+import {BrowserRouter, Switch, Route} from 'react-router-dom' 
+
+import {SignUpForm} from 'components/SignUpForm'
+import {SignInForm} from 'components/SignInForm'
+import {FirstPage} from 'components/FirstPage'
+import {Secrets} from 'components/Secrets'
+import './app.css'
 
 export const App = () => {
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
+    <BrowserRouter>
+      <main className='backgroundContainer'>
+        <Switch>
+          <Route path="/" exact>
+            <FirstPage/>
+          </Route>
+          <Route path="/users" exact>
+            <SignUpForm/>
+          </Route>
+          <Route path="/sessions" exact>
+            <SignInForm/>
+          </Route>
```

Comment:
These routes have nothing to do with the backend, so making them understandable to the user could be something to consider.  Consider:
* `/SignUp` instead of `/users`
* `/SignIn` instead of `/sessions`

The fetch URL is where the backend paths/urls are important